src/index.ts

```diff
@@ -1 +1,15 @@
 
+import logger, { type LoggerOptions } from 'pino'
+
+
+let cnf: LoggerOptions = {}
+if (!process.env.VERSION) {
+  cnf.transport = {
+    target: 'pino-pretty',
+    options: {
+      colorize: true,
+      singleLine: true,
+    },
+  }
+}
+export let log = logger(cnf)
```

Comment:
Quiero exportarlo como `logger`. La cuestión es que puede que te obligue a renombrar el otro.

```suggestion
export let logger = logger(cnf)
```